public/app/features/dashboard/state/DashboardModel.ts

```diff
@@ -173,6 +171,16 @@ export class DashboardModel implements TimeModel {
     this.ensurePanelsHaveIds();
     this.formatDate = this.formatDate.bind(this);
 
+    dispatch(
+      updateDashboard({
+        title: data.title ?? 'No Title',
+        liveNow: Boolean(data.liveNow),
+        graphTooltip: data.graphTooltip ?? DashboardCursorSync.Off,
+        description: data.description,
+        tags: data.tags ?? [],
+      })
+    );
+
```

Comment:
We create DashboardModel in a few instances (outside initDashboard) to leverage some of the defaults / getSaveModel features, this would then cause that to modify the global redux state.